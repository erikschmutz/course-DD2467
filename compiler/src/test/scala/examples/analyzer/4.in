let a = 1
let b = 1
let c = a + b

(*====================
val a : int
val b : int
val c : int
- :int
====================*)