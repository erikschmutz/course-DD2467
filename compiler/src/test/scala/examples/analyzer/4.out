AnalyzerResult(
    Integer(),
    Enviroment(
        List(
            ValEntry(
                Identifier(a),
                Integer()
            ),
            ValEntry(
                Identifier(b),
                Integer()
            ),
            ValEntry(
                Identifier(c),
                Integer()
            )
        )
    )
)